packages/scene-composer/src/utils/entityModelUtils/updateNodeEntity.ts

```diff
@@ -44,6 +44,16 @@ export const updateEntity = async (
     },
   };
 
+  if (node.parentRef) {
```

Comment:
I think this might need to be more robust?  If the node had a parentRef before but it's gone now we also need to do a parentEntityUpdte?  But also if the parentRef exists and hasn't changed then parentEntityUpdate is an extra API call that's redundant. Probably will require passing a little more info on old state to the function call though.

Reply:
It is true that adding extra redundant during the api call. However, it does not need change updateEntity parameters, which has other places to be called.

Reply:
will do a patch.